pkg/deploy/dev-workspace-config/dev_workspace_config.go

```diff
@@ -97,6 +97,8 @@ func updateWorkspaceConfig(cheCluster *chev2.CheCluster, operatorConfig *control
 	}
 
 	updateStartTimeout(operatorConfig, devEnvironments.StartTimeoutSeconds)
+
+	operatorConfig.Workspace.DeploymentStrategy = devEnvironments.DeploymentStrategy
```

Comment:
```suggestion
	operatorConfig.Workspace.DeploymentStrategy = utils.GetValue(devEnvironments.DeploymentStrategy, constants.DefaultDeploymentStrategy)
```
1. I suggest to remove any defaults from CheCluster CRD since DWO has its own ones.
2. Move default `Recreate` to `contants.go`

Reply:
Done, I removed the default from CheCluster CRD and moved the default value to constants.go

Reply:
I had to update the existing devworkspace-operator config tests, since now the default deployment strategy value is always used (programmatically, rather than from the CRD).

I had considered something like:

```go
if devEnvironments.DeploymentStrategy != "" {
operatorConfig.Workspace.DeploymentStrategy = v1.DeploymentStrategyType(utils.GetValue(string(devEnvironments.DeploymentStrategy), constants.DefaultDeploymentStrategy))
}
```

However, then removing the `devEnvironments.deploymentStrategy` value in the CheCluster CR would not reset the devworkspace-operator config deployment strategy back to its default value of `Recreate`. E.g. if you set `devEnvironments.deploymentStrategy: RollingUpdate`, then delete the `devEnvironments.deploymentStrategy` from the CheCluster CR, the devworkspace-operator config deployment strategy would stay with `RollingUpdate` (which is a non-default value).

Maybe there's a better way to go about this that I haven't considered. Any thoughts?

Reply:
@AObuchow 
Why not to keep
```
operatorConfig.Workspace.DeploymentStrategy = v1.DeploymentStrategyType(utils.GetValue(string(devEnvironments.DeploymentStrategy), constants.DefaultDeploymentStrategy))
```

Do you see any issues with that?

Reply:
@tolusha I'm fine with keeping it like that (though note it affects existing and future test cases as the expected devworkspace-operator configuration will always have a deploymentStrategy set).

Reply:
Correct, so I'm +1 for keeping the current approach.